### DISCUSSION TITLE
"Too many attempts" requesting users in a multiplayer lobby

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#29850] [Q&A]
The game could not load 3 users in the lobby which in turn made spectating not easy (they would load until a timeout, then i could see the remaining players). Everything worked fine after restarting the game.

I don't know if there is anything actionable here, but maybe you can find something useful, or prevent this from happening again.

![Screenshot_2](https://github.com/user-attachments/assets/d5500582-e0d8-47a0-90cd-58d096d7cba4)
![Screenshot_1](https://github.com/user-attachments/assets/9d2ecdf9-e101-4277-b805-adb09fde1664)
![Screenshot_3](https://github.com/user-attachments/assets/c7d614ba-aa30-4af5-8940-55445513ebf4)

[1725624788.network.log](https://github.com/user-attachments/files/16976551/1725624788.network.log)
[1725624788.runtime.log](https://github.com/user-attachments/files/16976552/1725624788.runtime.log)

**Comment by peppy** [comment]
Should be resolved via https://github.com/ppy/osu-web/pull/11506 and other related changes in recent days 🤞.
